README-DEV.rst

```diff
@@ -127,6 +127,20 @@ ignore their build and avoid any issues with their dependencies.
 
 See ``./configure --help`` for more information.
 
+Developing
+----------
+
+Formatting
+~~~~~~~~~~
+
+The ``erl`` files in ``src`` are formatted using erlfmt_. The checks are run
```

Comment:
I am not sure if it is a github render problem. There might be missing underscore here. I see it `erlfmt_` while I think you wanted `_erlfmt_`.

Reply:
Never mind I confused it with markdown.

Reply:
I almost never write rst so I tried to copy formatting from other parts of this file (and probably some googling happened, too).